### PR TITLE
Update pom.xml

### PR DESCRIPTION
**Pull request by jwnasambu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.openmrs.distro</groupId>
 	<artifactId>referenceapplication</artifactId>
-	<version>2.8.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (jwnasambu) [reply pom.xml]: @dkayiwa Thanks for creating time to check on my work

**Comment by dkayiwa** [comment]
@jwnasambu are you still working on this?

File: pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwnasambu) [reply pom.xml]: dkayiwa which one is meant to be worked so as to set up the uat-refapp.openmrs.org/

**Comment by dkayiwa** [comment]
Can you start by resolving the merge conflicts on this pull request?

**Comment by jwnasambu** [comment]
I have resolved the conflict

**Comment by jwnasambu** [comment]
dkayiwa apparently i have no conflict as you can see but all my checks have failed how can i resolve that?

**Comment by dkayiwa** [comment]
The pull request seems to have so many commits. Can you quash them into one?
If you are not able to squash them into one, you can find it easier to just do this:

1) Revert your changes
2) Switch to the master branch
3) Create a new branch
4) Put in the changes
5) Create a new pull request

**Pull request closed without merging**